### PR TITLE
Track nextjs part of the stack in sentry

### PR DESCRIPTION
**Pull request by razzeee**
This is basically just https://github.com/vercel/next.js/tree/canary/examples/with-sentry#create-and-configure-your-app done for our stack.

@barthalion We might want a different stack to track this to, to not mic python and typescript? But I have no idea, if that's the best thing we can do in sentry.

### CODE REVIEW

File: frontend/sentry.client.config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs"
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN
+
+Sentry.init({
+  dsn: SENTRY_DSN || "",
```
Comment (razzeee) [inline frontend/sentry.client.config.js:10]: In the created original `""` is filled with some sentry url, I choose not to commit that for now.

File: frontend/sentry.server.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs"
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN
+
+Sentry.init({
+  dsn: SENTRY_DSN || "",
```
Comment (razzeee) [inline frontend/sentry.server.config.js:10]: In the created original `""` is filled with some sentry url, I choose not to commit that for now.

File: frontend/sentry.properties [hunk 3]
```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=flathub
+defaults.project=prd-backend-v2
```
Comment (razzeee) [inline frontend/sentry.properties:3]: We probably want to change this.

Comment (barthalion) [reply frontend/sentry.properties:3]: Created a project named `frontend`.

**Comment by barthalion** [comment]
@razzeee I've set `NEXT_PUBLIC_SENTRY_DSN` both in repo secrets and Kubernetes. I will leave resolving conflicts to you as I wouldn't know what I'm doing.

**Comment by razzeee** [comment]
@barthalion rebased

**Pull request merged**